packages/patternfly-4/react-core/src/components/Progress/Progress.js

```diff
@@ -49,20 +50,21 @@ const defaultProps = {
 };
 
 class Progress extends Component {
-  constructor(props) {
```

Comment:
Great PR! But we do need to initialize the unique id in the constructors. Or at least have some other way of assigning unique id to the component. Props get evaluated before the component instances are created so if you have multiple of the same components on the page they will all have the same id otherwise.